management/connection.go

```diff
@@ -403,6 +406,8 @@ type ConnectionOptionsSMS struct {
 
 	OTP *ConnectionOptionsOTP `json:"totp,omitempty"`
 
+	AuthParams map[string]string `json:"authParams,omitempty"`
```

Comment:
How did you find out about this property? I have modeled the `ConnectionOptionsEmail` and `ConnectionOptionsSMS` connections by reverse engineering the dashboard... Is there any documentation on this?

Reply:
Only way to find this was to actually configure the Passwordless Email and SMS connections and then call the Management API to see what comes back.

Reply:
Ah, I see! Thanks for clarifying that.